game_event/views.py

```diff
@@ -63,3 +65,52 @@ def game_events(request):
         game_events = game_events.filter(time__gte=current_date)
 
     return render(request, 'game-events.html', {'game_events': game_events})
+
+
+def game_event(request, id):
+    event = get_object_or_404(GameEvent, pk=id)
+    player = Player.objects.get(user=request.user)
+    event_players = GameEventPlayer.objects.filter(game_event=event)
+    event_players = [{'first_name': entry.player.user.first_name, 'last_name': entry.player.user.last_name,
+                     'brings_ball': entry.ball_responsible} for entry in event_players]
```

Comment:
I'd refrain from using the same variable for holding different data types.
Try this instead:
```suggestion
    event_players = [
        {
            "first_name": entry.player.user.first_name,
            "last_name": entry.player.user.last_name,
            "brings_ball": entry.ball_responsible,
        }
        for entry in GameEventPlayer.objects.filter(game_event=event)
    ]
```

Reply:
more readable also, I'll change it as requested.